src/pages/Roadmap.tsx

```diff
@@ -0,0 +1,68 @@
+import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
+import { tsx } from '@dojo/framework/widget-core/tsx';
+
+import Card from '../widgets/card/Card';
+import CardHeader from '../widgets/card/CardHeader';
+import FontAwesomeIcon from '../widgets/icon/FontAwesomeIcon';
+import LocalPage from '../widgets/page/LocalPage';
+import Page from '../widgets/page/Page';
+
+import * as css from './Roadmap.m.css';
+
+interface TimelineEntry {
+	title: string;
+	date: string;
+	file: string;
+	released: boolean;
+}
+
+const timelineEntries: TimelineEntry[] = [
```

Comment:
I noticed that we're explicitly pulling this data based on this configuration, whereas the blog work uses convention to query a specific content folder. I feel like we should unify our approach in some way: either blogs should use explicit configuration, or roadmap (and other similar pages) should use convention to pull all markdown files in a given content location.

Reply:
Yeah I had thought of using your setup (specifically the meta data parsing), and probably will once we get it all merged together. This was the far easier way to get it up and running though. Ideally you would simply make a new file in the roadmap folder and it would order them by date automatically.